adev/src/app/core/services/version-manager.service.ts

```diff
@@ -60,14 +103,23 @@ export class VersionManager {
   }
 
   private getVersion(versionMode: VersionMode): string {
-    if (versionMode === 'stable') {
-      return 'v17';
-      // Temporarily commenting out till this works correctly
-      // return `v${VERSION.major}`;
+    if (versionMode === 'stable' || versionMode === 'deprecated') {
+      return `v${VERSION.major}`;
     }
     if (Number.isInteger(versionMode)) {
       return `v${versionMode}`;
     }
     return versionMode.toString();
   }
+
+  private getAdevDocsUrl(version: VersionMode | number): string {
```

Comment:
It looks like `number` is already a part of the `VersionMode` type.
```suggestion
  private getAdevDocsUrl(version: VersionMode): string {
```